packages/engine/src/ecs/classes/Engine.ts

```diff
@@ -34,44 +34,35 @@ import { Entity } from './Entity'
  * @author Josh, Vyacheslav, Gheric and the XREngine Team
  */
 export class Engine {
+  static instance: Engine
```

Comment:
What's the purpose of this instance?

Reply:
see https://github.com/XRFoundation/XREngine/issues/5834